### ISSUE TITLE
container doesn't get beyond "Looking for existence of /tmp/.container/10-redis-init"

### ISSUE THREAD

**Issue by joergmschulz** [tiredofit/docker-valkey#1]
### Summary
valkey doesn't start. 

### Steps to reproduce

in docker-compose:
```
nct-valkey:
    image: tiredofit/valkey:7
    container_name: nct-valkey
    volumes:
      - /data/faudin/nct-valkey:/data/db
    environment: 
      - CONTAINER_ENABLE_MONITORING=FALSE
      - CONTAINER_ENABLE_SCHEDULING=FALSE
      - DEBUG_MODE=TRUE
      - ENABLE_LOGS=FALSE
      - CONTAINER_NAME=nct-valkey
      - VALKEY_PASS=${REDIS_PASSWORD}
      - VALKEY_PORT=6381
    networks:
      services:
        ipv4_address: ${NCT_VALKEY_IP}
    restart: unless-stopped
```

### Relevant logs and/or screenshots
```
++ SOCKET_PERMISSIONS=777
nct-valkey  | + PROCESS_NAME=valkey
nct-valkey  | + check_container_initialized
nct-valkey  | + print_debug 'Checking to see if container initialization scripts have completed'
nct-valkey  | + output_off
nct-valkey  | + '[' true = true ']'
nct-valkey  | + set +x
nct-valkey  | 2024-07-02.20:34:29 [DEBUG] /etc/services.available/10-redis/run ** [valkey] Checking to see if container initialization scripts have completed
nct-valkey  | + output_off
nct-valkey  | + '[' true = true ']'
nct-valkey  | + set +x
nct-valkey  | + check_service_initialized init
nct-valkey  | + print_debug 'Checking to see if service has initialized'
nct-valkey  | + output_off
nct-valkey  | + '[' true = true ']'
nct-valkey  | + set +x
nct-valkey  | 2024-07-02.20:34:29 [DEBUG] /etc/services.available/10-redis/run ** [valkey] Checking to see if service has initialized
nct-valkey  | + output_off
nct-valkey  | + '[' true = true ']'
nct-valkey  | + set +x
nct-valkey  | 2024-07-02.20:34:30 [DEBUG] /etc/services.available/10-redis/run ** [valkey] Looking for existence of /tmp/.container/10-redis-init
nct-valkey  | + '[' '!' -f /tmp/.container/10-redis-init ']'
nct-valkey  | + sleep 1
nct-valkey  | + print_debug 'Looking for existence of /tmp/.container/10-redis-init'
nct-valkey  | + output_off
nct-valkey  | + '[' true = true ']'
nct-valkey  | + set +x
nct-valkey  | 2024-07-02.20:34:31 [DEBUG] /etc/services.available/10-redis/run ** [valkey] Looking for existence of /tmp/.container/10-redis-init
nct-valkey  | + '[' '!' -f /tmp/.container/10-redis-init ']'
nct-valkey  | + sleep 1
```

**Comment by tiredofit** [comment]
Woops! Still has some legacy wording in there. Let me do a once over and resolve immediately.

**Comment by tiredofit** [comment]
OK! Resolved with `7-2.19.0` or `7` or `latest` ... There was a script directory still named 10-redis which then forced to look for a function file with same name. Tested and `valkey-server` starts fine.

**Comment by joergmschulz** [comment]
Thanks a lot!

1113:M 02 Jul 2024 21:08:01.699 * Ready to accept connections tcp
1113:M 02 Jul 2024 21:08:01.699 * Ready to accept connections unix
